### ISSUE TITLE
[1.12.2] [BUG] Crash on Startup

### ISSUE THREAD

**Issue by H4mm3rb0y** [ForgeEssentials/ForgeEssentials#2472]
**Initial Steps**
- [X] I have updated to the latest build from Jenkins before reporting a bug
- [X] I have checked, if the bug was not already reported by using the [search](https://github.com/ForgeEssentials/ForgeEssentials/issues?q=).
- [x] Attached is my ForgeEssentials/modlist.txt, my latest.log, and my debug.log
   -> modlist.txt not available
- [x] Also, attached are the client logs (if applicable)

**Describe the bug**
When i start the server with the latest Version of FE it crashes at some point. I can't figue out why the server is crashing. The log does not seem to give me an answer.

**Desktop (please complete the following information):**
 - OS: Debian 10

**Additional context**

[debug.log](https://github.com/ForgeEssentials/ForgeEssentials/files/3880022/debug.log)
[latest.log](https://github.com/ForgeEssentials/ForgeEssentials/files/3880023/latest.log)
[crash-2019-11-22_15.43.36-server.txt](https://github.com/ForgeEssentials/ForgeEssentials/files/3880031/crash-2019-11-22_15.43.36-server.txt)

**Comment by JackCarterSmith** [comment]
Hi, can you post main.cfg and PlayerLogger.cfg from ForgeEssentials folder?

**Comment by H4mm3rb0y** [comment]
Sure, thanks for the fast answer.
I changed the file extension to txt due to github limitations. In the FE folder they have the cfg extension.

The h2-database file mentioned in the playerlogger.cfg wasn't created... could this be the problem?

[PlayerLogger.txt](https://github.com/ForgeEssentials/ForgeEssentials/files/3882473/PlayerLogger.txt)
[main.txt](https://github.com/ForgeEssentials/ForgeEssentials/files/3882474/main.txt)

**Comment by JackCarterSmith** [comment]
If you run your minecraft server on debian, it's possible that the server doesn't have write/read perms on the ForgeEssentials folder.
Can you check it with `ls -la` in the FE folder?

**Comment by H4mm3rb0y** [comment]
I have already checked the read/write permissions of the FE folder. FE created serveral files before the crash without any problem. So i don't think this could be the case, will check it again anyways when i'm back home.

**Comment by H4mm3rb0y** [comment]
As i already predicted above the rights are all set correctly. :/

**Comment by spacebuilder2020** [comment]
Check to see if your 'lib' folder in ./ForgeEssentials/lib exists and that it has 19 jar files in it.  Based on your log, player logger is unable to find a library that it needs.

**Comment by H4mm3rb0y** [comment]
> Check to see if your 'lib' folder in ./ForgeEssentials/lib exists and that it has 19 jar files in it. Based on your log, player logger is unable to find a library that it needs.

yep, 19 libs are in the folder, all accessible by the user running the server.

**Comment by H4mm3rb0y** [comment]
Hey,

Today i tried if the error is present anymore. Sadly i encountered the same error i mentioned above.
I enabled the debug mode in the main.cfg and reuploaded a [debug.log](https://gist.github.com/H4mm3rb0y/4a2fce38ed4b4fa97c6185781f126ba8) for you. If i disable the module PlayerLogger in the main.cfg the server seems to start. I will now do some testing but at this point the only issue seems to be the PlayerLogger module causing this error

At this point i don't know if this has to do with another mod running on the server due to the huge amount on mods we are using. Perhabs this [modlist](https://gist.github.com/H4mm3rb0y/92c668f4f8ebdf6d94139c83e6407b76) helps you.

With best regards
H4mm3r

**Comment by JackCarterSmith** [comment]
Did you try to create a second empty server with only FE? 
Do you have a crash-report file with during the last crash?

**Comment by Zethalion** [comment]
Is this still an issue @H4mm3rb0y
